scripts/unit_test.sh

```diff
@@ -83,7 +83,7 @@ echo '===================================='
 echo '=== Installing Gem Dependencies ===='
 echo '===================================='
 set +e
-podman exec -u 0 "$TEST_CONTAINER_ID" /bin/bash -c 'microdnf install -y $DEV_DEPS'
```

Comment:
@marleystipich2 is there a reason to drop the `-y` argument?

Reply:
Nope, not necessary actually.. Trying to get the PR check to pass 🙈